src/api/commands/user-managment.ts

```diff
@@ -29,20 +30,25 @@ import { Scim2Constants } from "../constants/scim2.0-constants";
  * @param  {jsonbody} reqBody - request body with user profile informations
  * @param  {boolean} failOnStatusCode- Whether to fail on response codes other than 2xx and 3xx
  * */
-Cypress.Commands.add("createUserWithScim", ( host,authrzUserName,authrzPassword,reqBody,
-    failOnStatusCode = true): Cypress.Chainable<any> => {
+Cypress.Commands.add("createUserViaAPI", (host, username, password, reqBody, grantType, authType,
```

Comment:
Lets add types.

ex:


```suggestion
Cypress.Commands.add("createUserViaAPI", (host: string, ...,
```

Reply:
Fixed